### ISSUE TITLE
Ads on REG

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#2395]
[https://app.zenhub.com/files/3918508/fb3b01f4-7dac-4f62-a93f-c8156c0a6b4e/download](https://app.zenhub.com/files/3918508/fb3b01f4-7dac-4f62-a93f-c8156c0a6b4e/download)

**Comment by Captainkirkdawson** [comment]
@PatReynolds @richpomfret @Vino-S I have discovered the cause and solution to the strange ads appearing on FreeREG. It is not MM code. By inspecting the running code I observed that these were google auto ads
![ad-auto.JPG](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/53103397-366e-4721-b06f-3f7bc8c680d2)
On inspection I found that the AdSense setting for FreeREG had been changed to Auto ads which meant that google was displaying whatever ads it wanted and where ever it wanted.
Do not know when or why it was enabled but in my mind it is inappropriate. I have changed the setting back to no auto ads; it will take an hr or 2 and they should disappear.
While looking at AdSense setting I notice that FreeBMD also has the auto ads on; I would suggest that this should be turned off
